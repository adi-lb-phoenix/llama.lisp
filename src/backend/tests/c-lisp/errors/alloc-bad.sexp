(c-lisp
    (define ((main void))
        (declare arr (ptr float))
        (set arr (alloc 3))))
