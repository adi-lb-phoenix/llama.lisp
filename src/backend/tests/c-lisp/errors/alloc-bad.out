Error in statement: ['set', 'arr', ['alloc', 3]]
CodegenError: Bad alloc expression: ['alloc', 3]
